### ISSUE TITLE
Careers page

### ISSUE THREAD

**Issue by tammycamp** [actionfactory/stronghold.co#7]
Here is the Stack template page for careers - http://stronghold.co/page-careers.html

The copy will be very similiar to the following:

https://www.cockroachlabs.com/careers/

**Comment by LAKnoKAL** [comment]
@tammycamp can you please explain how Careers page will looks, should i design it on my own and make it looks similar to `cockroachlabs` ?

**Comment by tammycamp** [comment]
Yes! Can you design in like cockroach Labs?  Do something similar but keep the "current openings" part the same on the Stack template @LAKnoKAL

**Comment by tammycamp** [comment]
Hi @LAKnoKAL 

Can we make some changes?

As noted above, can you please keep the "current openings" section the same as the original template?

Also, keep the fonts the same as well?  For example, the font is not BOLD in the header in the original template.  We would like it to stay that way.  The BOLD fonts look to chunky.  If you could restyle to be the originals in the template that would be great.

Thanks!
